ansi/ansi.go

```diff
@@ -1,10 +1,8 @@
 package ansi
 
-import "regexp"
-
-var ansiEscape = regexp.MustCompile(`\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])`)
+import "github.com/charmbracelet/x/ansi"
```

Comment:
I would add a name to this import.

Because importing ansi in ansi package doesn't sound a good idea